### PR TITLE
python3-trimesh: update to 3.8.1.

### PR DESCRIPTION
**Pull request by karl-nilsson**

### CODE REVIEW

**Comment by Piraty** [comment]
xlint: `Place checkdepends= after depends=`

**Comment by karl-nilsson** [comment]
@Piraty fixed, thanks

**Comment by Chocimier** [comment]
> TypeError: don't know how to make test from: <trimesh.exceptions.ExceptionModule object at 0x7f76eb5ce520>

passes for you?

**Comment by Piraty** [comment]
not for me

**Comment by karl-nilsson** [comment]
> passes for you?

No, but the newest changes should (mostly) pass.
I'm working on packaging rtree

**Comment by Piraty** [comment]
tests still don't pass, require [a few more modules](https://github.com/mikedh/trimesh/blob/5b3c5d4/setup.py#L31-L47)

**Comment by karl-nilsson** [comment]
> tests still don't pass, require [a few more modules](https://github.com/mikedh/trimesh/blob/5b3c5d4/setup.py#L31-L47)

Strictly speaking, In order to pass all tests, all dependencies must be included:
https://github.com/mikedh/trimesh/issues/590#issuecomment-549457839

What's an acceptable solution: package all of requirements_easy, then have pytest do a conditional import? Or just package all requirements?

**Comment by Piraty** [comment]
if it's not feasable to package the missing modules (and they're not required at runtime at all), i'd say skip `do_check` (but leave a comment).

**Comment by karl-nilsson** [comment]
@Piraty I was able to package the remaining dependencies, and make check seems to work (albeit with warnings).
Let me know what you think.

File: srcpkgs/python3-pycollada/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-pycollada'
+pkgname=python3-pycollada
+version=0.7.1
+revision=1
+archs=noarch
+wrksrc="pycollada-${version}"
+#create_wrksrc=yes
```
Comment (Piraty) [inline srcpkgs/python3-pycollada/template:7]: please remove

**Review by Piraty — CHANGES_REQUESTED** [review]
looks good!

**Pull request closed without merging**